covert/idstore.py

```diff
@@ -0,0 +1,93 @@
+from covert import passphrase, pubkey
+from covert.blockstream import encrypt_file, decrypt_file
+from covert.archive import Archive
+from pathlib import Path
+import mmap
+import subprocess
+import os
+
+def idfilename():
```

Comment:
In my experience, moving configuration path handling to a separate module that collects the assumptions about path name handling, is benefitial. Especially Path.home() / ".covert" is probably something that could change later. Perhaps due to an environment variable override or something else. Anyway, all path names could be generated in one module. Most importantly the covert program's base directory Path.home() / ".covert" should be non-redundant.

Reply:
Both this and the permissions, quite possibly other handling as well is set to be refactored into util functions (taking care of OS-dependent/configurable config dirs, according to xdg specification etc). But this PR is a draft which will certainly go through many revisions prior to that.